#![allow(unused_imports)]
use std::error::Error;

use adventofcode2022::{
    day1, day10, day11, day12, day13, day14, day15, day16, day17, day2, day3, day4, day5, day6,
    day7, day7b, day8, day9,
};

fn main() -> Result<(), Box<dyn Error>> {
    println!("Advent of Code 2022");
    // day1::main()?;
    // day2::main()?;
    // day3::main()?;
    // day4::main()?;
    // day5::main()?;
    // day6::main();
    // day7::main()?;
    // day7b::main()?;
    // day8::main()?;
    // day9::main()?;
    // day10::main()?;
    // day11::main()?;
    // day12::main()?;
    // day13::main()?;
    // day14::main()?;
    // day15::main()?;
    // day16::main()?;
    day17::main()?;
    // day18::main()?;
    // day19::main()?;
    // day20::main()?;
    // day21::main()?;
    // day22::main()?;
    // day23::main()?;
    // day24::main()?;
    // day25::main()?;

    Ok(())
}
